packages/js/components/src/experimental-select-tree-control/select-tree.tsx

```diff
@@ -38,6 +40,8 @@ export const SelectTree = function SelectTree( {
 	initialInputValue,
 	onInputChange,
 	shouldShowCreateButton,
+	isClearButtonVisible = false,
```

Comment:
Instead of this, I would rather prefer `allowClearing`.

Reply:
I like the idea of renaming the prop, but since it's a `boolean` I would start it with an `is`. I renamed the prop to `isClearingAllowed` in the commit [2a12d5a](https://github.com/woocommerce/woocommerce/pull/49036/commits/2a12d5a9f2cbb25345bb2a44511e850eb1f11f2d)